### ISSUE TITLE
[x11] events don't work in testnative

### ISSUE THREAD

**Issue by madebr** [libsdl-org/SDL#8111]
For example, the `SDL_EVENT_QUIT` event is never triggered (checked with a debugger).
This is the complete stdout after clicking on the `x`:
```
INFO: Creating native window for x11 driver
X Error of failed request:  BadWindow (invalid Window parameter)
  Major opcode of failed request:  146 ()
  Minor opcode of failed request:  1
  Resource id in failed request:  0x1200001
  Serial number of failed request:  4303
  Current serial number in output stream:  4304
```

**Comment by slouken** [comment]
Yeah, foreign windows aren't managed by SDL in any way. This is a common request though, so I'll put it on the SDL 3.0 milestone.

**Comment by slouken** [comment]
So we've enabled mouse and keyboard events. The close button still doesn't work though.

@icculus, even if I register the WM protocols, we still don't get ClientMessage events for the window. Can you take a look?

**Comment by slouken** [comment]
We are scoping work for the SDL 3.2.0 release, so please let us know if this is a showstopper for you.
